dlrs/main/classes/RollupController.cls

```diff
@@ -28,8 +28,7 @@
  * Handles the Manage Trigger and Calculate Custom Buttons
  **/
 public with sharing class RollupController {
-  
-  public static String API_VERSION = '58.0';
+  public static String DEF_API_VERSION = '58.0';
```

Comment:
How about
```
@testvisible
private static final String MDAPI_API_VERSION_FALLBACK_VALUE
//...
public String package_xml_api_version
```

mostly I am aiming at telling names.

Reply:
I didn't take the full recommended name but I did make changes that I think respected the recommendations.